### DISCUSSION TITLE
taro针对跨团队(多工程、多仓库）或者说多工作区，最后来整合主包、分包的模式，有例子可以参考吗。是否可以直接参考原生混合模式。

### DISCUSSION THREAD

**Discussion by journey-xt** [NervJS/taro#14481] [Q & A]
### 这个特性解决了什么问题？
1、跨团队维护小程序
2、多业务线整合小程序

### 这个 API 长什么样？
有一个例子提供思路

**Comment by lexmin0412** [comment]
我写的一个插件，之前有在公司项目中应用过，可以看看是不是你想要的

https://github.com/lexmin0412/taro-plugin-internal

**Comment by journey-xt** [comment]
> 我写的一个插件，之前有在公司项目中应用过，可以看看是不是你想要的
> 
> https://github.com/lexmin0412/taro-plugin-internal

谢谢，看了下，这个思路不够好。

我感觉应该是按原声混合模式。分为  main 、sub。min拉取的是sub的产出作为自己的分包。主包完全不用管sub的依赖。这种形式才能更好的协作。

**Comment by agileago** [comment]
@journey-xt 你看看这个  https://github.com/hellof2e/dora

**Comment by lexmin0412** [comment]
> > 我写的一个插件，之前有在公司项目中应用过，可以看看是不是你想要的
> > https://github.com/lexmin0412/taro-plugin-internal
> 
> 谢谢，看了下，这个思路不够好。
> 
> 我感觉应该是按原声混合模式。分为 main 、sub。min拉取的是sub的产出作为自己的分包。主包完全不用管sub的依赖。这种形式才能更好的协作。

那确实，按照你的应用场景，原生混合模式才是合适的。

**Comment by journey-xt** [comment]
> @journey-xt 你看看这个 https://github.com/hellof2e/dora

哈哈，今天在 微信看到了 dora的文章。发现是一个。真是巧了。谢谢大佬。
